keyboards/yetis/rules.mk

```diff
@@ -0,0 +1,2 @@
+CONVERT_TO = elite_pi
+PIN_COMPATIBLE = elite_c
```

Comment:
This not the way to set this.  You should set the processor as RP2040, and use the GP## pin names

Reply:
done